### DISCUSSION TITLE
Bug in Playlist

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#29815] [Ideas] [answered]
I think there's a bug in playlist where there's only one player in the leaderboard even there's 7 people who played in it.

I don't know if this is really a bug and if this is already fixed in the latest update.

![Screenshot_2024-09-06-14-20-16-459_sh ppy osulazer](https://github.com/user-attachments/assets/262c29d0-8923-45cb-9741-87ac5c2206ef)

**Comment by Joehuu** [comment, marked as answer]
The users on the top right are participants (i.e. they played but not necessarily passed / submitted a score).
